tensorflow/core/kernels/cwise_op_leakyrelu_bf16.cc

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#define EIGEN_USE_THREADS
+
+#include "tensorflow/core/kernels/cwise_ops_common.h"
+#include "tensorflow/core/kernels/cwise_ops.h"
+#include "tensorflow/core/framework/op_kernel.h"
+
+#define _USE_MATH_DEFINES
+#include <cmath>
+#include <functional>
+#include <type_traits>
+
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+#include "tensorflow/core/framework/bounds_check.h"
+#include "tensorflow/core/framework/numeric_types.h"
+#include "tensorflow/core/framework/tensor_types.h"
+
+// Keeping all new leakyrelu changes in 1 file.
+// This is similar to changes in cwise_ops.h
+namespace Eigen {
+namespace internal {
+
+template <typename Scalar>
+struct leakyrelu_op {
```

Comment:
```suggestion
class leakyrelu_op {
 public:
```

Reply:
Hi @cantonios : changing struct to class gave some error, so i removed the trailing underscore  and private. 
Now it seems to be in sync with the convention here : https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
We don't typically have private members in structs.  But sure, I'll see if this passes the lint test.